CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# v3.8.1 (Thu Feb 13 2025)
```

Comment:
Not sure why various version bumps have appeared in this PR. Same thing that happened in that survey banner PR.